### PR TITLE
Hub-737: Tear down short-hub abtest and keep the variant

### PR DESCRIPTION
**Pull request by jkunle**
**What**
We need to remove the short hub AB test and keep the variant.
https://govukverify.atlassian.net/browse/HUB-737
see: https://github.com/alphagov/verify-frontend-federation-config/pull/184
see: https://github.com/alphagov/verify-acceptance-tests/pull/106

**Why**

The variant won! We've set it to 100% so all users are seeing it, but now we need to do the cleanup.

**Description**
Current control artefacts i.e. controllers, views was renamed, the variant is then re-named to the control, the Controllers and views that are associated with the **long hub journey**  below has been removed in favour of the short-hub journey.
This is easier to review commit by commit

image

image

The Redirect to warning controller  below has been removed

image

Throttling functionality has been remove, there may still be some artefacts of it left in the config

The about pages ie. the educational pages for the LOA1 now uses the short-hub combined about page,
and now use single controller and view.

The old idp rules have been removed in favour of the short idp rules.

All traces of ABTest experiments have been removed from controller specs

### CODE REVIEW

File: spec/models/select_documents_form_spec.rb [hunk 1]
```diff
@@ -1,128 +0,0 @@
-require "spec_helper"
```
Comment (Wynndow) [inline spec/models/select_documents_form_spec.rb:1]: Deleting this spec file means that `app/models/select_documents_form.rb` now doesn't have any tests. I know the model is simpler but could you add something to cover it?

File: app/constraints/select_route.rb [hunk 2]
```diff
@@ -30,6 +28,8 @@ def cookie_matches_experiment?(request)
     end
   end
 
+private
```
Comment (Wynndow) [inline app/constraints/select_route.rb:31]: `cookie_matches_experiment?` only seems to be used in this  class, so could stay private?

Comment (Wynndow) [inline app/constraints/select_route.rb:31]: `cookie_matches_experiment?` only seems to be used in this  class, so could stay private?

File: config/locales/cy.yml [hunk 3]
```diff
@@ -184,25 +185,24 @@ cy:
...
-          <li>ydych dros 20 oed</li>
-          <li>oes gennych o leiaf un ddogfen hunaniaeth â llun, o unrhyw wlad</li>
+          <li>have a UK address</li>
+          <li>are over 20</li>
+          <li>have at least one photo identity document, from any country</li>
           </ul>
-        button: Defnyddiwch GOV.UK Verify
-      cannot_verify: Beth i'w wneud os na allwch ddefnyddio GOV.UK Verify
+        button: "Use GOV.UK Verify"
+      cannot_verify: What to do if you cannot use GOV.UK Verify
```
Comment (Wynndow) [inline config/locales/cy.yml:205]: This content seems to have become English - should probably stay as Welsh

**Review by Wynndow — CHANGES_REQUESTED** [review]
I don't have much experience with the AB stuff but the changes seem fine as far as I can tell. Only a few comments on some minor stuff - the translation stuff should probably be fixed up though.

File: spec/models/select_documents_form_spec.rb (hunk 1 above)
Comment (jkunle) [reply spec/models/select_documents_form_spec.rb:1]: Thanks

File: config/locales/cy.yml (hunk 3 above)
Comment (jkunle) [reply config/locales/cy.yml:205]: Many thanks for spotting this

**Review by Wynndow — APPROVED** [review]

**Review by JohnWattsGDS — APPROVED** [review]

**Pull request merged**